librz/core/cmd/cmd_analysis.c

```diff
@@ -5037,7 +5037,7 @@ static void cmd_analysis_abt(RzCore *core, const char *input) {
 			RzList *path;
 			RzListIter *pathi;
 			RzListIter *bbi;
-			rz_cons_printf("f orip=`dr?PC`\n");
+			rz_cons_printf("f+ orip @ `drq PC`\n");
```

Comment:
what's the difference with `f`?

Reply:
`f+` adds the flag to the same offset if there are already flags, `f` - doesn't. Before that `f` and `f+` behaved similarly, I changed behavior to be more logical and consistent.